tests/ceph_installer/test_cephadm.py

```diff
@@ -144,6 +148,11 @@ def run(ceph_cluster: Ceph, **kwargs) -> int:
             if not isvalid:
                 LOG.error("Log file validation failure")
                 return 1
+        if config.get("verify_log_rotate"):
+            isverified = validate_log_rotate(cephadm)
+            if not isverified:
+                LOG.error("Log rotate validation failure")
+                return 1
```

Comment:
This can be rewritten as -
```
        if config.get("verify_log_rotate") and validate_log_rotate(cephadm):
                LOG.error("Log rotate validation failure")
                return 1
```

Reply:
Done.